src/adminserver/handlers/router.go

```diff
@@ -27,5 +27,6 @@ func newRouter() http.Handler {
 	r.HandleFunc("/api/configurations", api.UpdateCfgs).Methods("PUT")
 	r.HandleFunc("/api/configurations/reset", api.ResetCfgs).Methods("POST")
 	r.HandleFunc("/api/systeminfo/capacity", api.Capacity).Methods("GET")
+	r.HandleFunc("/api/v1/configurations/ping", api.Ping).Methods("GET")
```

Comment:
let's not add version to a specific API.
And the `ping` shouldn't be under `configuration`